### ISSUE TITLE
Bug in Atom.io

### ISSUE THREAD

**Issue by YongHee-Kim** [timholy/ProgressMeter.jl#45]
Progress meter in [Atom.io](https://atom.io/) looks like this
[1Gtest... 20%|████                |  ETA: 0:00:04[K

adding one more condition to printover function fixed it 

```

function printover(io::IO, s::AbstractString, color::Symbol = :color_normal)
    if isdefined(Main, :IJulia) || isdefined(Main, :ESS) || isdefined(Main, :Atom)
        print(io, "\r" * s)
    else
       ......

```

**Comment by timholy** [comment]
If you'd like, you can "get credit" for this discovery by browsing to that file https://github.com/timholy/ProgressMeter.jl/blob/master/src/ProgressMeter.jl, clicking the little pencil icon, making the edit, and submitting as a pull request. Your change will be automatically tested to make sure it doesn't break anything.

**Comment by YongHee-Kim** [comment]
Didn't knew how to do it. 
Thanks for fantastic packages, I've been using it everyday :)

**Comment by garrison** [comment]
Fixed in #46.
